src/components/views/right_panel/RoomHeaderButtons.tsx

```diff
@@ -44,6 +46,12 @@ const ROOM_INFO_PHASES = [
     RightPanelPhases.EncryptionPanel,
     RightPanelPhases.Room3pidMemberInfo,
 ];
+const UnreadIndicator = ({ className }) => {
```

Comment:
need to define types on `className`

Reply:
I am not sure If the changes I did is what you requested here. Let me know if you expected sth. else